docs/basic-reader-and-writer.md

```diff
@@ -366,4 +366,42 @@ altered and the set of Streams being read has therefore been changed.
 So that's it.  The simple HelloWorldReader loops, reading Events from a Stream
 until there are no more Events, and then the application terminates.
 
- 
+## ReaderGroup Notifications
+
+ReaderGroup api supports different types of notifications. The supported
```

Comment:
Replace:

ReaderGroup api supports different types of notifications. The supported notifications are

with


The `ReaderGroup` api supports different types of notifications. Currently, we have one type implemented, but we plan to add more over time. The type we currently support is the following:

Reply:
done.